presto-release-tools/src/main/java/com/facebook/presto/release/git/GitRepositoryModule.java

```diff
@@ -63,8 +64,9 @@ public GitRepositoryProvider(Class<? extends Annotation> annotation, String repo
         @Override
         protected GitRepository get(Injector injector, Class<? extends Annotation> annotation)
         {
-            FileRepositoryConfig config = injector.getInstance(Key.get(FileRepositoryConfig.class, annotation));
-            return GitRepository.fromFile(repositoryName, config);
+            GitRepositoryConfig repositoryConfig = injector.getInstance(Key.get(GitRepositoryConfig.class, annotation));
+            GitConfig gitConfig = injector.getInstance(Key.get(GitConfig.class));
+            return GitRepository.create(repositoryName, repositoryConfig, gitConfig);
```

Comment:
Static import?

Reply:
I think we do static import in most cases but not all cases. Whether to static import depends on if the static method name is specific and descriptive enough. This should be the case for most utility-like methods, but the names of factory constructor are usually very general, and thus we should not static import in those cases.